scipy/stats/tests/test_distributions.py

```diff
@@ -2381,6 +2381,34 @@ def test_logcdf(self):
                         stats.norm.logsf(np.log(x2-mu)/sigma))
 
 
+# Columns are x, a, b, cdf, sf.
+# The values of cdf and sf were computed using mpmath.
+_beta_cdf_sf = np.array([
+    [1e-20, 3.0, 0.5, 3.1249999999999994e-61, 1.0],
+    [1e-20, 0.5, 3.0, 1.875e-10, 0.9999999998125],
+    [0.999, 2, 10, 1.0, 1.0990000000000097e-29],
+    [0.9999999999999994, 3.0, 0.5, 0.9999999558234914, 4.417650858175538e-08],
+    [0.9999999999999994, 0.5, 3.0, 1.0, 5.345529420184392e-47],
+])
+
+# Columns are p, a, b, ppf.
+# The p column was computed using mpmath.
+_beta_ppf = np.array([
+    [2.5e-21, 3.0, 0.5, 1.999999949999998e-7],
+    [0.9999999407309943, 3.0, 0.5, 0.999999999999999],
+    [2.6516504294495528e-08, 0.5, 3.0, 2e-16],
+])
+
+# Columns are p, a, b, isf.
+# The p column was computed using mpmath.
+_beta_isf = np.array([
+    [0.99981250000125, 0.5, 3.0, 1.0e-8],
+    [2.0000030000132007e-17, 0.5, 3.0, 0.999996],
+    [5.930192380246625e-07, 3.0, 0.5, 0.9999999999999],
```

Comment:
This is one of the tests that should highlight the improvement of the PR branch, but it passes on either master or this PR branch with exactly 0 error. I'd suggest taking a few 9s of the end of the `isf` so there is some precision left for comparison.

Reply:
Yes, this and many others pass in master.  For those shape parameters, the version in master works reasonably well with small p inputs.  I could just remove this one, since it is not very interesting.  It is there because I was including known values for extreme inputs, whether or not they passed on master.  Only four of the new tests actually fail with the master branch.